grype/db/v6/cvss.go

```diff
@@ -0,0 +1,37 @@
+package v6
+
+import (
+	"github.com/anchore/grype/grype/vulnerability"
+)
+
+func toCvss(severities ...Severity) []vulnerability.Cvss {
+	//nolint:prealloc
+	var out []vulnerability.Cvss
+	for _, sev := range severities {
+		switch sev.Scheme {
+		case SeveritySchemeCVSS:
+		default:
+			// not a CVSS score
+			continue
+		}
+		score, ok := sev.Value.(CVSSSeverity)
+		if !ok {
+			// not a CVSS score
+			continue
+		}
+		out = append(out, vulnerability.Cvss{
+			Source:  sev.Source,
+			Type:    string(sev.Scheme),
+			Version: score.Version,
+			Vector:  score.Vector,
+			Metrics: vulnerability.CvssMetrics{
+				// FIXME: where do these metrics come from?
```

Comment:
these have not been used in the past and we should clean this up around grype 1.0

Reply:
actually, there might be a change, I'll get back to you